### PR TITLE
Updated About text and Plugin Logo

### PR DESCRIPTION
**Pull request by n0rdlicht**
Renames the Plugin to align with Frictionless Data wording. Changes in About text to include links to Gemeindescan project and cividi.

### CODE REVIEW

File: GemeindescanExporter/metadata.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [general]
-name=GemeindescanExporter
-description=Gemeindescan Exporter plugin
-about=Plugin to export data and styles to <a href="https://gemeindescan.ch/de/">gemeindescan</a>
-version=0.0.1
+name=SpatialDataPackageExporter
+description=Spatial Datapackage Exporter plugin
+about=Plugin to export data and styles to a Spatial Data Package compatible with <a href="https://bitbucket.org/cividi/gemeindescan-webui">Gemeindescan</a>.
+version=0.0.2
```
Comment (Joonalai) [inline GemeindescanExporter/metadata.txt:5]: CD process changes version number automatically when making release, you don't have to change this.

**Review by Joonalai — COMMENTED** [review]
There are still some mentions of Gemeindescan in the plugin. I can refactor those to be more fitting if you want?

File: GemeindescanExporter/metadata.txt (hunk 1 above)
Comment (n0rdlicht) [reply GemeindescanExporter/metadata.txt:5]: :+1: thanks, will leave it at 0.0.1

**Comment by Joonalai** [comment]
@n0rdlicht would it make sense to refactor all names and folders as well?

**Comment by n0rdlicht** [comment]
> @n0rdlicht would it make sense to refactor all names and folders as well?

I wouldn't do that just yet. Will get back to you on this and submit this draft as actual pull request and include the final decision on this in the comments.

**Comment by Joonalai** [comment]
> > @n0rdlicht would it make sense to refactor all names and folders as well?
> 
> I wouldn't do that just yet. Will get back to you on this and submit this draft as actual pull request and include the final decision on this in the comments.

Alright, it makes sense.

**Comment by loleg** [comment]
- I would call it "Spatial Data Export" instead of exporter, just for readability.
- It would be nice to have a web page like https://docs.gemeindescan.ch/ which is just a readthedocs.io instead of pointing to bitbucket
- An initial public release should be 0.1.0, not 0.0.1
- Have you checked the full list of compliance requests on https://plugins.qgis.org/ ?

**Comment by tjukanovt** [comment]
@loleg IMHO calling it _Spatial Data Export_ could maybe make sense from the perspective of Frictionless Data, but not really from the QGIS plugin repo and QGIS community. There are multiple plugins doing spatial data export in one form or another so maybe having the frictionless data maybe in the name somehow? _FSD Export_ (i.e. Frictionless Spatial Data)?

Also still about the license and QGIS plugins compliance: I think if we want to publish the plugin in the QGIS plugins repo, it should be GPL 2 or higher as discussed [here](http://blog.qgis.org/2016/05/29/licensing-requirements-for-qgis-plugins/). As this is the stance of QGIS community and it is also enforced (e.g. on Twitter and the mailing list).

**Comment by loleg** [comment]
:heavy_check_mark:  Agree [Spatial Data Package](https://research.okfn.org/spatial-data-package-investigation/) Export would make more sense.

:heavy_check_mark: I'm fine with the licensing compliance like this, but it will probably continue to be a point of discussion, and should be documented.

**Comment by Joonalai** [comment]
>     * An initial public release should be 0.1.0, not 0.0.1
I agree, the 0.0.1 was not meant to be a *public* release.
>     * Have you checked the full list of compliance requests on https://plugins.qgis.org/ ?
Yes we have and the plugin fulfills the requirements. Maybe there could be more descriptive tags in `metadata.txt`?

**Comment by n0rdlicht** [comment]
Regarding the naming: Going with „Spatial Data Package Export“ makes sense from my view. @Joonalai do you want to do a full rename based on this? Then we can make a 0.1.0 release together with moving the repo over to github.com/cividi. If yes I would open an issue for that and turn this PR into only finalizing the about text.

**Comment by Joonalai** [comment]
@n0rdlicht yes, sounds good. It would be great if you could provide the logo icon here as well. The best place would be `gemeindescanExporter/resources/icons/icon.png`.

**Pull request merged**